mobile/src/main/res/values-de/strings.xml

```diff
@@ -15,78 +15,78 @@
     <string name="app_discovery">Starte Suche</string>
     <!-- Main menu items -->
     <string name="mainmenu_openhab_preferences">Einstellungen</string>
-    <string name="mainmenu_openhab_selectsitemap">Wähle Sitemap</string>
-    <string name="mainmenu_openhab_writetag">Schreibe NFC Tag</string>
-    <string name="mainmenu_openhab_info">openHAB Info</string>
-    <string name="mainmenu_openhab_clearcache">Lösche Bildercache</string>
+    <string name="mainmenu_openhab_selectsitemap">Sitemap-Auswahl</string>
```

Comment:
I saw that is was wrong, but didnt want to make an extra pr, when you are working on this. 'Standard-Sitemap auswählen' sounds good